pypika/enums.py

```diff
@@ -58,9 +58,12 @@ class JoinType(Enum):
     cross = "CROSS"
 
 
-class UnionType(Enum):
-    distinct = ""
-    all = " ALL"
+class SetOperations(Enum):
```

Comment:
To be consistent with other Enums it probably makes sense to rename this to the singular SetOperation.

Reply:
will do

Reply:
@gl3nn Resolved the above comment